contracts/core/Dispatcher.sol

```diff
@@ -474,7 +477,11 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, ReentrancyGuard, IDi
      * @param timeoutTimestamp The timestamp, in seconds after the unix epoch, after which the packet times out if it
      * has not been received.
      */
-    function sendPacket(bytes32 channelId, bytes calldata packet, uint64 timeoutTimestamp) external nonReentrant {
+    function sendPacket(bytes32 channelId, bytes calldata packet, uint64 timeoutTimestamp)
```

Comment:
nit: update the docs with the new returned value?

Reply:
ah yes, added - good catch!